lib/src/flex/Flex.tsx

```diff
@@ -2,45 +2,12 @@ import React from "react";
 import styled from "styled-components";
 import FlexPropsType from "./types";
 
-export default function Flex({
-  direction = "row",
-  wrap = "nowrap",
-  justifyContent = "flex-start",
-  alignItems = "stretch",
-  alignContent = "normal",
-  gap = "0",
-  order = 0,
-  grow = 0,
-  shrink = 1,
-  basis = "auto",
-  alignSelf = "auto",
-  as,
-  children,
-}: FlexPropsType): JSX.Element {
-  return (
-    <FlexContainer
-      direction={direction}
-      wrap={wrap}
-      justifyContent={justifyContent}
-      alignItems={alignItems}
-      alignContent={alignContent}
-      order={order}
-      grow={grow}
-      shrink={shrink}
-      basis={basis}
-      alignSelf={alignSelf}
-      gap={gap}
-      as={as}
-    >
-      {children}
-    </FlexContainer>
-  );
-}
+const DxcFlex = ({ children, ...props }: FlexPropsType): JSX.Element => <Flex {...props}>{children}</Flex>;
```

Comment:
You need the default values.

Reply:
Agree! I've just restored them (I though we were using the default value of css anyways but in some cases we aren't).